### ISSUE TITLE
Decreased Mid-Upper Arm Circumference 

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5896]
**Preferred term label:** Decreased Mid-Upper Arm Circumference 

**Synonyms** Decreased MUAC

**Definition (free text, please give PubMed ID)** The measurement of the mid-upper arm is used to assess the nutritional status of the person. When the circumference halfway down the upper arm is decreased this could mean the person is malnourished. It has also been shown as an accurate way to measure the body weight of children., PMID: 27831914

**Parent term (use hpo.jax.org/app)** Abnormality of the upper arm HP:0001454

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
